.teamcity/Gradle_Util_Performance/buildTypes/AdHocPerformanceTestCoordinatorLinux.kt

```diff
@@ -0,0 +1,44 @@
+package Gradle_Util_Performance.buildTypes
+
+import common.Os
+import common.applyPerformanceTestSettings
+import common.buildToolGradleParameters
+import common.builtInRemoteBuildCacheNode
+import common.checkCleanM2
+import common.compileAllDependency
+import common.distributedPerformanceTestParameters
+import common.gradleWrapper
+import common.performanceTestCommandLine
+import jetbrains.buildServer.configs.kotlin.v2018_2.BuildType
+
+object AdHocPerformanceTestCoordinatorLinux : BuildType({
+    uuid = "a28ced77-77d1-41fd-bc3b-fe9c9016bf7b"
+    id("Gradle_Util_Performance_PerformanceTestCoordinatorLinux")
+    name = "AdHoc Performance Test Coordinator - Linux"
+
+    applyPerformanceTestSettings(os = Os.linux, timeout = 420)
+
+    maxRunningBuilds = 2
+
+    params {
+        param("performance.baselines", "defaults")
+    }
+
+    steps {
```

Comment:
I saw several repetitions of 

```
gradleWrapper {
            name = "GRADLE_RUNNER"
            tasks = ""
            gradleParams = (...)
```

Why don't reuse `gradleRunnerStep`?

Reply:
`gradleRunnerStep` uses the whole model type, which I don't have available for the ad-hoc tasks. It also doesn't provide much convenience (only the name of the step). What code would you have expected to be re-used?